setup/borg/cli.md

```diff
@@ -228,9 +260,56 @@ consistency:
     check_last: 3
 ```
 
+{% endtab %}
+
+{% tab borgmatic_config Borgmatic 1.8.x %}
```

Comment:
Not `1.8.x`, but `>1.8....`. Same above for any versions below.

Reply:
this works?

```
{% tab borgmatic_config Borgmatic >= 1.8 %}
```

and

```
{% tab borgmatic_config Borgmatic <= 1.7 %}
```

Reply:
Yes, that's fine. 👍